pkg/controller/operators/suite_test.go

```diff
@@ -88,14 +88,14 @@ var _ = BeforeSuite(func() {
 	useExisting := false
 	testEnv = &envtest.Environment{
 		UseExistingCluster: &useExisting,
-		CRDs: []client.Object{
-			crds.CatalogSource(),
-			crds.ClusterServiceVersion(),
-			crds.InstallPlan(),
-			crds.Subscription(),
-			crds.OperatorGroup(),
-			crds.Operator(),
-			crds.OperatorCondition(),
+		CRDs: []apiextensionsv1.CustomResourceDefinition{
```

Comment:
Is this something we should move to across the board when using controller-runtime moving forward?